src/components/Header/Header.scss

```diff
@@ -1,9 +1,150 @@
 @import '~style/variables';
 
-.wrapper {
-  border: 1px solid blue;
+header {
+  display: flex;
+  flex-direction: column;
+  background-color: $color-primary--500;
+  color: $color-white;
+
+  > * {
```

Comment:
Minor: *generally* selectors like this don't scale well. It becomes easy to accidentally apply rules in places you don't mean. My preferred approach is to only ever target by class name (i.e. no wildcards, no tag names). It's also a theoretical performance hit because the browser matches CSS selectors right to left. So will match all components first, then filter that down to "all components that have a parent element with a `header` class.

But I suspect there's no real world impact on a site this size, and as for selectors this is probably not a real thing to worry about either. Feel free to ignore.